src/checks/create-is-single.js

```diff
@@ -11,7 +11,7 @@ const createIsSingle = {
   name: 'createIsSingle',
   shouldRun: isCreate,
   run: (method, results) => {
```

Comment:
It isn't used here, but it's used in other checks, like this one: https://github.com/zapier/zapier-platform-core/blob/master/src/checks/is-search.js#L3